coderd/prometheusmetrics/collector.go

```diff
@@ -49,6 +56,13 @@ func (v *CachedGaugeVec) Collect(ch chan<- prometheus.Metric) {
 }
 
 func (v *CachedGaugeVec) WithLabelValues(operation VectorOperation, value float64, labelValues ...string) {
+	switch operation {
+	case VectorOperationAdd:
+	case VectorOperationSet:
+	default:
```

Comment:
I usually prefer this. But it does not matter.
```golang
switch operation {
	case VectorOperationAdd, VectorOperationSet:
	default:
}
```

Reply:
https://github.com/coder/coder/pull/7085